### PR TITLE
Update macos.md

### PR DESCRIPTION
**Pull request by kentbull**
Document both the -v and -vv build logging commands.

This would have saved my team a day or two if this were in the documentation

### CODE REVIEW

**Comment by ndonkoHenri** [comment]
Thanks for your PR. 

A section concerning verbosity was [present](https://github.com/flet-dev/website/commit/8f485567d0675190ee93340dea47bbd69224bf38#diff-7e0eea1b3f89f6c99fd83d5196e74d4f435d24bdb7ce49d5a41f72bbdf227b56R353-R357), but seems like it was mistakenly removed when updating the docs.

Can you add it [here](https://flet.dev/docs/publish#logging) (before logging) instead?

File: docs/publish/macos.md [hunk 1]
```diff
@@ -11,4 +11,20 @@ The command can be run on macOS only.
...
\ No newline at end of file
+Creates a macOS application bundle from your Flet app.
+
+### Specify output directory
+
+If you specify the build directory then the directory acts like a cache, speeding up subsequent builds.
+
+```bash
+flet build -o ./build macos
+```
```
Comment (ndonkoHenri) [inline docs/publish/macos.md:22]: A default output directory is used when one is not specified: `build/macos` when using `flet build macos`

Comment (kentbull) [reply docs/publish/macos.md:22]: I had just been using `-o ./build`. I didn't know about the default. That can be removed from these changes.

**Comment by kentbull** [comment]
I'll just close this PR since I can't update the original code. I think it would be worth mentioning the `-vv` option on each individual platform's publishing page as well.

**Comment by kentbull** [comment]
See this new PR: https://github.com/flet-dev/website/pull/285

**Pull request closed without merging**